README.md

```diff
@@ -13,26 +13,29 @@ In order to set your environment up to run the code here, first install all requ
 pip install -r requirements.txt
 ```
 
+Rename example.env to .env and edit the variables appropriately.
+MODEL_TYPE supports LlamaCpp or GPT4All
+
 Then, download the 2 models and place them in a folder called `./models`:
 - LLM: default to [ggml-gpt4all-j-v1.3-groovy.bin](https://gpt4all.io/models/ggml-gpt4all-j-v1.3-groovy.bin). If you prefer a different GPT4All-J compatible model, just download it and reference it in `privateGPT.py`.
```

Comment:
This line should also be updated to point to .env

Reply:
Slipped past me, got it